packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -5133,6 +5133,981 @@ curl -v -X GET \
+
+##### Response parameters
+
+Returns the deactivated [Client Secret](#client-secret-response-object)
+
+##### Request example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/pkcs10" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/apps/0oad5lTSBOMUBOBVVQSC/credentials/secrets/ocsap6nz61rKdsoyOY0h7/lifecycle/deactivate"
+```
+
+##### Response example
+
+```json
+{
+  "id": "ocsap6nz61rKdsoyOY0h7",
+  "client_secret": "3vimrC5Yv6bSDJzrUdLEYvkf9ElwUeWdndO5nhYp",
+  "secret_hash": "0xbeefbeefbeefbeefbeefbeef",
+  "created": "2022-01-01T04:22:27.000Z",
+  "status": "INACTIVE",
+  "_links": {
+    "activate": {
+      "href": "https://${yourOktaDomain}/api/v1/apps/0oad5lTSBOMUBOBVVQSC/credentials/secrets/ocsap6nz61rKdsoyOY0h7/lifecycle/activate",
+      "hints": {
+        "allow": [
+          "POST"
+        ]
+      }
+    },
+    "delete": {
+      "href": "https://${yourOktaDomain}/api/v1/apps/0oad5lTSBOMUBOBVVQSC/credentials/secrets/ocsap6nz61rKdsoyOY0h7",
+      "hints": {
+        "allow": [
+          "DELETE"
+        ]
+      }
+    }
+  }
+}
+```
+If a secret isn't found for the `secretId`, you receive an error response.
+
+```json
+{
+  "errorCode": "E0000007",
+  "errorSummary": "Not found: Resource not found: 1234 (OAuth2ClientSecretMediated)",
+  "errorLink": "E0000007",
+  "errorId": "oaeMHrsk2WLTACvPU5T7yQ4yw"
+}
+```
+
+If the application has only one active client secret, and if a deactivate request is sent for that client secret, you receive an error response.
+
+```json
+{
+  "errorCode": "E0000001",
+  "errorSummary": "Api validation failed: OAuth2ClientSecretMediated",
+  "errorLink": "E0000001",
+  "errorId": "oaeMHrsk2WLTACvPU5T7yQ4yw",
+  "errorCauses": [
+    {
+      "errorSummary": "You can't deactivate the only active client secret."
+    }
+  ]
+}
+```
+
+### Delete a client secret
+
+<ApiOperation method="delete" url="/api/v1/apps/${applicationId}/credentials/secrets/{secretId}"/>
+
+Deletes a specific client secret by `secretId`
+
+##### Request parameters
+
+| Parameter     | Description                                                                     | Param Type | DataType                                      | Required | Default |
+| ------------- | ------------------------------------------------------------------------------- | ---------- | --------------------------------------------- | -------- | ------- |
+| applicationId | Unique key of an [Application](#application-object)                             | URL        | String                                        | TRUE     |         |
+| secretId      | Unique key of a [Client Secret](#client-secret-response-object)             | URL        | String                                        | TRUE     |         |
+
+##### Response parameters
+
+An empty JSON object `{}`
```

Comment:
If we return a 204 without content, then there's no response body, i.e. no empty json object.

Reply:
Followed https://developer.okta.com/docs/reference/api/apps/#request-parameters-7

Reply:
Then that's incorrect too - a 204 has no response body, so we should fix those docs as well if you'd want to stay consistent in the docs.

Reply:
updated